operators/validate_nid.go

```diff
@@ -110,9 +114,9 @@ func nidUs(nid string) bool {
 
 	sequence := true
 	equals := true
-	prev, _ := strconv.Atoi(string(nid[0]))
+	prev := int(nid[0] - '0')
```

Comment:
On reread it's pretty mysterious :) Consider a small `digitToInt` helper function

Reply:
Done.